src/server/game/Spells/Auras/SpellAuras.cpp

```diff
@@ -1423,12 +1423,17 @@ void Aura::HandleAuraSpecificMods(AuraApplication const* aurApp, Unit* caster, b
                 if (!caster)
                     break;
                 // Ice barrier - dispel/absorb remove
-                if (removeMode == AURA_REMOVE_BY_ENEMY_SPELL && GetSpellInfo()->SpellFamilyFlags[1] & 0x1)
+                if ((removeMode == AURA_REMOVE_BY_ENEMY_SPELL) &&
+                    (GetSpellInfo()->SpellFamilyFlags[EFFECT_1] & 0x00000001) &&
```

Comment:
How should it be done then?
All other usages of SpellFamilyFlags seem to do the same